README.md

```diff
@@ -0,0 +1,265 @@
+# Lights Off!
+
+Lights Off! is a rendition of the original handheld game "Lights Out!", manufactured by Tiger Electronics. If you haven't played it before, you may well find Lights Off! to be a very addictive puzzle game.
+
+![LightsOff Screenshot](LightsOff.png)
+
+### How to Play
+
+The objective is to turn all the lights off by pushing the fewest number of buttons. Easier said than done. ;^)
+
+All 1050 puzzles from the original ("Classic") game have been included, but arranged a bit differently. The puzzles have been grouped into puzzle packs under the 'Puzzle Packs' menu. The original game's main puzzles are in the "Classic" pack. The rest are included from the manual for the original game and have been grouped by difficulty, with "Six Move Puzzles" being the easiest and "Fifteen Move Puzzles" being the hardest. Beginners are advised to attempt the Classic pack before trying the others. Note that you may switch from one puzzle pack to another and not lose the progress you have made. You may also re-play any puzzle you have already solved by choosing it from the 'Levels' menu. The Classic pack features 50 puzzles, and the others host 100 each.
+
+When you press a button, you turn it off or on, just like a light switch. Additionally, any button above, below, or beside the button you have pushed is also switched on or off.
+
+### Winning
+
+You must solve a puzzle within 10 moves of the minimum required. This means that you must solve the every puzzle in the 'Six Move Puzzles' pack within 16 moves. For the Classic pack, the requirements are listed below:
+
+<table border="1">
+
+<tbody>
+
+<tr>
+
+<td>Level</td>
+
+<td>Moves Required</td>
+
+<td>Level</td>
+
+<td>Moves Required</td>
+
+</tr>
+
+<tr>
+
+<td>Levels 1-5</td>
+
+<td>6</td>
+
+<td>Levels 25-30</td>
+
+<td>11</td>
+
+</tr>
+
+<tr>
+
+<td>Levels 6-10</td>
+
+<td>7</td>
+
+<td>Levels 30-35</td>
+
+<td>12</td>
+
+</tr>
+
+<tr>
+
+<td>Levels 11-15</td>
+
+<td>8</td>
+
+<td>Levels 35-40</td>
+
+<td>13</td>
+
+</tr>
+
+<tr>
+
+<td>Levels 16-20</td>
+
+<td>9</td>
+
+<td>Levels 40-45</td>
+
+<td>14</td>
+
+</tr>
+
+<tr>
+
+<td>Levels 20-25</td>
+
+<td>10</td>
+
+<td>Levels 45-50</td>
+
+<td>15</td>
+
+</tr>
+
+</tbody>
+
+</table>
+
+### How to Solve Classic Level 1
+
+Classic Level one looks like the table below, with lit buttons in blue:
+
+<table border="1">
+
+<tbody>
+
+<tr>
+
+<td>1</td>
+
+<td>2</td>
+
+<td>3</td>
+
+<td>4</td>
+
+<td>5</td>
+
+</tr>
+
+<tr>
+
+<td>6</td>
+
+<td>7</td>
+
+<td>8</td>
+
+<td>9</td>
+
+<td>10</td>
+
+</tr>
+
+<tr>
+
+<td>**<font color="#0000ff">11</font>**</td>
```

Comment:
Markdown doesn't seem to like formatting in tables. The "**" doesn't bold things. Remove the "**" in all tables, we'll have to rely on the colours.

Reply:
It seems that git strips HTML formatting code for security reasons.